manage.py

```diff
@@ -312,3 +314,12 @@ def config(config_key):
 @app.cli.command()
 def set_celery_beat_healthy():
     return celery_beat_health_check()
+
+
+@app.cli.command()
+def healthcheck():
+    """Calls the healthcheck API"""
+    result = requests.get(
+        url_for('check')
+    )
+    print(result.text)
```

Comment:
Nice addition! Can you return a non-`0` exit code if the response isn't OK?

Reply:
Done! Let me know if this is what you were looking for. If not, I'll update.